nixos/modules/programs/qt5ct.nix

```diff
@@ -1,31 +1,11 @@
-{ config, lib, pkgs, ... }:
+{ lib, ... }:
 
 with lib;
 
 {
   meta.maintainers = [ maintainers.romildo ];
```

Comment:
I think it would be better to remove yourself from maintainers here and move it to the `qt5` module instead if you're still interested on it.

Reply:
Do you mean add myself as maintainer of the `qt5` module?